pandas/tests/io/conftest.py

```diff
@@ -14,6 +14,12 @@ def jsonl_file(datapath):
     return datapath('io', 'parser', 'data', 'items.jsonl')
 
 
+@pytest.fixture
+def unicode_series_file(datapath):
```

Comment:
Hmm not so sure about this being a dedicated fixture - what's the thought in doing this instead of just resolving `datapath` in the functions that need it?

Reply:
sorted.